hikari/impl/rest.py

```diff
@@ -745,18 +745,39 @@ async def _request(
                         start = time.monotonic()
 
                     # Make the request.
-                    response = await live_attributes.still_alive().client_session.request(
-                        compiled_route.method,
-                        url,
-                        headers=headers,
-                        params=query,
-                        json=json,
-                        data=form,
-                        allow_redirects=self._http_settings.max_redirects is not None,
-                        max_redirects=self._http_settings.max_redirects,
-                        proxy=self._proxy_settings.url,
-                        proxy_headers=self._proxy_settings.all_headers,
-                    )
+                    try:
```

Comment:
The try except should be taken out of here and put before the `with` (in the base of this PR, a try already exists there, but it is not there in the current state of master, so a rebase will first be necessary)